src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1791,49 +1758,37 @@
     short-summary: Create a peering.
     examples:
         - name: Create a virtual network peering between virtual networks in the same region
-          text: >
+          text: |
             az network vnet create --name myVnet1 --resource-group myResourceGroup --location eastus --address-prefix 10.0.0.0/16
-            \\n\\n az network vnet create --name myVnet2 --resource-group myResourceGroup --location eastus --address-prefix 10.1.0.0/16
-            \\n\\n vnet1Id=$(az network vnet show --resource-group myResourceGroup--name myVnet1 --query id --out tsv)
-            \\n\\n vnet2Id=$(az network vnet show --resource-group myResourceGroup --name myVnet2 --query id --out tsv)
-            \\n\\n az network vnet peering create --name myVnet1ToMyVnet2 --resource-group myResourceGroup --vnet-name myVnet1 --remote-vnet-id $vnet2Id --allow-vnet-access
-            \\n\\n az network vnet peering create --name myVnet2ToMyVnet1 --resource-group myResourceGroup --vnet-name myVnet2 --remote-vnet-id $vnet1Id --allow-vnet-access
+            az network vnet create --name myVnet2 --resource-group myResourceGroup --location eastus --address-prefix 10.1.0.0/16
+            vnet1Id=$(az network vnet show --resource-group myResourceGroup--name myVnet1 --query id --out tsv)
+            vnet2Id=$(az network vnet show --resource-group myResourceGroup --name myVnet2 --query id --out tsv)
+            az network vnet peering create --name myVnet1ToMyVnet2 --resource-group myResourceGroup --vnet-name myVnet1 --remote-vnet-id $vnet2Id --allow-vnet-access
+             az network vnet peering create --name myVnet2ToMyVnet1 --resource-group myResourceGroup --vnet-name myVnet2 --remote-vnet-id $vnet1Id --allow-vnet-access
 
         - name: Create a virtual network peering between virtual networks in different regions
```

Comment:
We should probably call out that this example is 100% bash specific. It will not work on CMD.exe.

Reply:
As determined by myself and @JasonRShaver the standard for all CLI examples is `bash`. If there must be CMD/Powershell-specific examples, they will need to be documented separately from the CLI inline help. In particular we would need engineering support for:

1. Two different example YAML tags
2. Web rendering support for these tags from OPS
3. CLI engineering support for OS detection to determine which example to display
4. _Potentially_ per-platform customized help where environment variables or system values are referenced

This is a non-trivial amount of work. If telemetry indicates that CMD/Powershell inline examples are a necessity than we can discuss.